src/services/email/ses-email-templates/index.js

```diff
@@ -138,15 +137,14 @@ details by clicking on its ID number.</li>
 <p>
 <br><b>State or territory:</b> {{territory}}
 <br><b>Name:</b> {{submitterName}}
-<br><b>Email:</b> {{submitterEmail}}
-<br><b>1915(b) {{actionType}} Number:</b> {{id}}</b>
-<br><b>Waiver Authority:</b> {{authority}}
+<br><b>Email Address:</b> {{submitterEmail}}
+<br><b>Initial Waiver Number:</b> {{id}}
```

Comment:
I think this was a bit too far because not all 1915(b) submissions are Initial Waiver. There are also Renewal and Amendment Waivers. I suspected this and tested and found this does indeed bleed into the other waiver types:
<img width="1100" alt="image" src="https://github.com/Enterprise-CMCS/macpro-mako/assets/99831795/db73c03e-d3bf-4a80-a36b-edf8dbabb326">